src/test/java/kitchenpos/products/Fixtures.java

```diff
@@ -1,26 +1,48 @@
 package kitchenpos.products;
 
-import kitchenpos.products.model.Product;
 
+import java.lang.reflect.Field;
 import java.math.BigDecimal;
+import kitchenpos.products.model.ProductRequest;
+import kitchenpos.products.tobe.domain.Price;
+import kitchenpos.products.tobe.domain.Product;
 
 public class Fixtures {
     public static final Long FRIED_CHICKEN_ID = 1L;
     public static final Long SEASONED_CHICKEN_ID = 2L;
 
-    public static Product friedChicken() {
-        final Product product = new Product();
-        product.setId(FRIED_CHICKEN_ID);
+    public static ProductRequest friedChickenRequest() {
+        final ProductRequest product = new ProductRequest();
         product.setName("후라이드");
         product.setPrice(BigDecimal.valueOf(16_000L));
         return product;
     }
 
-    public static Product seasonedChicken() {
-        final Product product = new Product();
-        product.setId(SEASONED_CHICKEN_ID);
+    public static Product friedChicken() {
+        return newProduct("후라이드", FRIED_CHICKEN_ID);
+    }
+
+    public static ProductRequest seasonedChickenRequest() {
+        final ProductRequest product = new ProductRequest();
         product.setName("양념치킨");
         product.setPrice(BigDecimal.valueOf(16_000L));
         return product;
     }
+
+    public static Product seasonedChicken() {
+        return newProduct("양념치킨", SEASONED_CHICKEN_ID);
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        return new Product(SEASONED_CHICKEN_ID, "양념치킨", BigDecimal.valueOf(16_000L));
```